addons/website_hr_recruitment/static/src/scss/website_hr_recruitment.scss

```diff
@@ -37,3 +102,48 @@
         overflow: hidden;
     }
 }
+
+.o_jobs_topbar_filters {
+    .dropdown-toggle {
+        border: $border-width solid gray('400');
+        @include o-bg-color(gray('white'), $with-extras: false);
+        @include border-radius($dropdown-border-radius);
+        @include hover-focus {
```

Comment:
Is all this really needed ? Can't we use the default design instead?

Moreover this approach:
- generate inconsistencies across modules
- overrides the theme design (even worst, may lead to a mix of both designs)
- overrides variable customisations written by the user using the HTML editor
- ...

I'm aware that currently other modules customize these component, but we should standardise rather than jeopardise even more.